.github/CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 # See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 /src/libs/brand*.js @DataBiosphere/terra-cobranding
 /src/libs/colors.js @DataBiosphere/terra-cobranding
+/src/pages/LandingPage.js @DataBiosphere/terra-cobranding
```

Comment:
I would recommend you relax the syntax on these so you would pick up unit tests if/when they get written.

BTW, have you found a way to test this before it is submitted? I also have a PR up, but am not sure how to verify the syntax is correct (though iterating on it if needed doesn't seem like a big deal)-- https://github.com/DataBiosphere/terra-ui/pull/3473

Reply:
> BTW, have you found a way to test this before it is submitted? I also have a PR up, but am not sure how to verify the syntax is correct (though iterating on it if needed doesn't seem like a big deal)

In the "Files changed" tab of the PR, GitHub shows a warning if there are syntax errors or an indication that the file is valid.
![Screen Shot 2022-10-31 at 9 24 56 AM](https://user-images.githubusercontent.com/1156625/199018503-da1e8ddc-63fe-4a21-b51f-168d51a293b7.png)

Also when viewing the file: https://github.com/DataBiosphere/terra-ui/blob/dev/.github/CODEOWNERS

https://github.blog/changelog/2022-02-17-codeowners-improvements-syntax-errors-preview-of-who-will-be-requested-and-more/

Reply:
I saw that, but I meant checking that my wildcard syntax really is picking up the files that I wanted.

Reply:
Aha! You can view the file in the PR branch and if it is configured with code owners, it will have a little shield icon. Hovering over that shield shows the code owners.

https://github.com/DataBiosphere/terra-ui/blob/ky_codeowners_landingpage/src/pages/LandingPage.js
![Screen Shot 2022-10-31 at 9 44 12 AM](https://user-images.githubusercontent.com/1156625/199022571-700d76ee-78df-4d4b-8205-393e97c87fa3.png)

Reply:
Oh wow - hawk eyes right there 🤩